src/controllers/AuthController.js

```diff
@@ -31,6 +33,24 @@ const forgotPassword = async (request, response) => {
   }
 };
 
+/**
+ *
+ * @name logOut
+ * @param {object} req
+ * @param {object} res
+ * @returns {json} - json
+ */
+const logOut = (req, res) => {
+  const token = req.headers.authorization;
+  const decodedData = jwt.verify(token, process.env.SECRET_KEY);
+  const expTime = decodedData.exp;
```

Comment:
Since the token was already decoded in the `verifyToken` middleware, why not just attach the token and the expiry date to the request object and then use them here, instead of having to repeat ourselves by decoding the token here again

Reply:
Alright. I will refactor it